deployment/ccip/deploy.go

```diff
@@ -93,7 +93,7 @@ type ContractDeploy[C Contracts] struct {
 
 // TODO: pull up to general deployment pkg somehow
 // without exposing all product specific contracts?
-func deployContract[C Contracts](
+func DeployContract[C Contracts](
```

Comment:
This got changed to help break the cycle, as only one thing in integration-tests needs it, so we can avoid moving the whole structure there.